lua/core/options.lua

```diff
@@ -133,3 +133,4 @@ local function load_options()
 end
 
 load_options()
+local _ = pcall(require, "user.options")
```

Comment:
Yet it is necessary to load user options:
```suggestion
local ok, user_options = pcall(require, "user.options")
if ok then
	for name, value in pairs(user_options) do
		vim.o[name] = value
	end
end
```

Those options can also be merged earlier on L124-126.